thorvg-android/src/main/java/com/lottiefiles/dotlottie/core/widget/DotLottieAnimation.kt

```diff
@@ -134,19 +135,15 @@ class DotLottieAnimation @JvmOverloads constructor(
     }
 
     fun play() {
-        mLottieDrawable?.start()
-    }
-
-    fun setBackgroundColor(color: String) {
-        mLottieDrawable?.setBackgroundColor(color)
+        mLottieDrawable?.play()
     }
```

Comment:
setBackgroundColor is not longer in the dotlottie core kotlin bindings, but you can use `setConfig` to override the backgroundColor